api/src/main/java/org/pmiops/workbench/billing/FreeTierBillingService.java

```diff
@@ -140,11 +140,18 @@ private boolean expiredByCost(final DbUser user, final double currentCost) {
     return compareCosts(currentCost, getUserFreeTierDollarLimit(user)) > 0;
   }
 
-  private void deactivateUserWorkspaces(final DbUser user) {
-    final Set<DbWorkspace> toDeactivate = workspaceDao.findAllByCreator(user);
-    for (final DbWorkspace workspace : toDeactivate) {
-      workspaceDao.updateBillingStatus(workspace.getWorkspaceId(), BillingStatus.INACTIVE);
-    }
+  // TODO: move to DbWorkspace?  RW-5107
+  private boolean isFreeTier(final DbWorkspace workspace) {
+    return workspace
+        .getBillingAccountName()
+        .equals(workbenchConfigProvider.get().billing.freeTierBillingAccountName());
+  }
+
+  private void setActiveStatusForCreatedWorkspaces(final DbUser user, final BillingStatus status) {
```

Comment:
I hate to ask for more characters in this verbose method name, but seems like this should probably specify `FreeTierWorkspaces`